src/vs/workbench/services/hover/browser/hoverWidget.ts

```diff
@@ -128,6 +128,9 @@ export class HoverWidget extends Widget {
 			}
 		});
 
+		// Hide when the window loses focus
+		this._register(dom.addDisposableListener(window, 'blur', () => this.dispose()));
```

Comment:
You might want to use `dom.EventType.BLUR`

Reply:
Why would this be more bytes? 
I would think that using a shared string literal save memory. Otherwise `blur` will exists many times in the string literal buffer. That said I don't know how sophisticated the VM manages string literals.

Reply:
Browsers chromium, probably Firefox) only hold a single copy of each unique string. The extra bytes I mean in our js bundle which I think has its name unmangled as it's exported.